### PR TITLE
*: Update to Proemtheus v2.5.0 as default

### PR DESCRIPTION
**Pull request by metalmatze**
This will set Prometheus v2.5.0 as default version for Prometheus in the kube-prometheus stack.

### CODE REVIEW

**Comment by brancz** [comment]
Could you also bump the jsonnet dependency and re-generate?

File: Documentation/compatibility.md [hunk 1]
```diff
@@ -39,4 +39,3 @@ The versions of Prometheus compatible to be run with the Prometheus Operator are
 * v2.4.1
 * v2.4.2
 * v2.4.3
-* v2.5.0
```
Comment (brancz) [inline Documentation/compatibility.md]: that doesn't look right

**Comment by krasi-georgiev** [comment]
that was quick. Thanks team!

cc @mrsiano

**Pull request merged**